pages/01.metakube/01.release-notes/default.en.md

```diff
@@ -10,6 +10,17 @@ page-toc:
 <!-- markdownlint-disable MD036 -->
 ## July 2019
 
+**2019-07-22**
+
+* Kubernetes versions 1.14.4, 1.12.10, 1.13.8, 1.15.0 and 1.15.1 are now available
+* Updated [Helm](https://www.helm.sh/) to [2.14.2](https://github.com/helm/helm/releases/tag/v2.14.2)
+* On Openstack, the scheduler is now aware that only 25 additional volumes may be attached to a node. It will make sure that pods with mounted volumes are only scheduled on nodes can support them.
```

Comment:
`are only scheduled on nodes can support them` .. this should be `...on nodes that support them` (or alike)